web/index.html

```diff
@@ -539,11 +585,14 @@ <h3>Used in</h3>
                 addElementScrollPage: function () {
                     this.loadedElements *= 2;
                 },
-                smoothScrollTo: function (position = 0) {
+                smoothScrollTop: () => {
+                    if (window.pageYOffset > 500) {
```

Comment:
Updated smooth scroll to jump to nearly the top instantly before the smooth scrolling action, which is not noticable to the user, it just looks a lot cleaner